koku/cost_models/serializers.py

```diff
@@ -54,6 +55,44 @@ class MarkupSerializer(serializers.Serializer):
     unit = serializers.ChoiceField(choices=MARKUP_CHOICES, required=False)
 
 
+class DistributionSerializer(BaseSerializer):
+    """Serializer for distribution options"""
+
+    DISTRIBUTION_OPTIONS = ("distribute_type", "worker_cost", "platform_cost")
+
+    distribute_type = serializers.ChoiceField(choices=metric_constants.DISTRIBUTION_CHOICES, required=False)
+    platform_cost = serializers.BooleanField(required=False)
+    worker_cost = serializers.BooleanField(required=False)
+
+    def validate_distribute_type(self, distribute_type):
+        """Run validation for distribute type."""
+        distrib_choice_list = [choice[0] for choice in metric_constants.DISTRIBUTION_CHOICES]
+        if distribute_type not in distrib_choice_list:
+            error_msg = f"{distribute_type} is an invaild distribution type"
+            raise serializers.ValidationError(error_msg)
+        return distribute_type
+
+    def validate(self, data):
+        """Run validation for distribution options."""
+
+        if not any(ele in data.keys() for ele in self.DISTRIBUTION_OPTIONS):
```

Comment:
The membership test on a dict operates on the keys.
```suggestion
        if not any(ele in data for ele in self.DISTRIBUTION_OPTIONS):
```